go.mod

```diff
@@ -24,6 +24,7 @@ require (
 	github.com/esimonov/ifshort v1.0.4
 	github.com/fatih/color v1.13.0
 	github.com/fzipp/gocyclo v0.5.0
+	github.com/GaijinEntertainment/go-exhaustruct v0.0.0-20220320194800-c73fb34375ae
```

Comment:
Can you create a tag?
We use dependabot to update linters, so tags are required.

Reply:
done

Reply:
you also have to update the PR :wink:

Reply:
Yet not updatable since `go mod` says that there is no such version, so waiting for now🤷‍♂️

Reply:
I updated the go.mod

Reply:
Just updated.
should be fine now

Reply:
FYI the module is available immediately you don't need to wait.
The fact to call the module adds the module to the Go proxy.

Reply:
I had like `http 423: gone` during `go mod` execution, unable to recall exactly

Reply:
have create several time the same tag?